torchvision/prototype/datasets/_builtin/eurosat.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict, List, Tuple
+
+from torchdata.datapipes.iter import IterDataPipe, Mapper
+from torchvision.prototype.datasets.utils import Dataset, DatasetConfig, DatasetInfo, HttpResource, OnlineResource
+from torchvision.prototype.datasets.utils._internal import hint_sharding, hint_shuffling
+from torchvision.prototype.features import EncodedImage, Label
+
+
+class EuroSAT(Dataset):
+    def _make_info(self) -> DatasetInfo:
+        return DatasetInfo(
+            "EuroSAT",
+            homepage="https://github.com/phelber/eurosat",
+            categories=(
+                "AnnualCrop",
+                "Forest",
+                "HerbaceousVegetation",
+                "Highway",
+                "Industrial," "Pasture",
+                "PermanentCrop",
+                "Residential",
+                "River",
+                "SeaLake",
+            ),
+        )
+
+    def resources(self, config: DatasetConfig) -> List[OnlineResource]:
+        url_root = "https://madm.dfki.de/files/sentinel"
+        data = HttpResource(
+            f"{url_root}/EuroSAT.zip",
+            sha256="8ebea626349354c5328b142b96d0430e647051f26efc2dc974c843f25ecf70bd",
+        )
+        return [data]
```

Comment:
Using an URL root makes the use case of multiple files from the same server more concise. With only one URL, it is not needed

```suggestion
        return [
            HttpResource(
                f"https://madm.dfki.de/files/sentinel/EuroSAT.zip",
                sha256="8ebea626349354c5328b142b96d0430e647051f26efc2dc974c843f25ecf70bd",
            )
        ]
```